.gitignore

```diff
@@ -163,3 +163,5 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+data/
```

Comment:
Can you change this to checkpoints instead of data? I want to avoid collisions with the data module in the package.

Reply:
Fixed